app/frontend/src/components/QuestionInput/QuestionInput.tsx

```diff
@@ -65,7 +65,7 @@ export const QuestionInput = ({ onSend, disabled, placeholder, clearOnSend, init
     };
 
     const disableRequiredAccessControl = requireLogin && !loggedIn;
-    const sendQuestionDisabled = disabled || !question.trim() || requireLogin;
+    const sendQuestionDisabled = disabled || !question.trim();
```

Comment:
Hm, I wonder if it would be more prudent to have `|| disableRequiredAccessControl`? Would that also resolve the bug, since it should only be true when not logged in.